templates/root/index.html

```diff
@@ -25,8 +25,15 @@
 <div id="content">
 <div class="box">
 	<p>
-To install Haiku, we offer <strong>stable</strong> release builds as well as <strong>nightly</strong>
-builds which are continually built by a cluster of dedicated machines from the <a href="http://cgit.haiku-os.org/haiku/">latest Haiku source code</a>.
+To install Haiku, we offer <strong>stable</strong> release builds, as well as <strong>nightly</strong>
+builds. The <strong>nightly builds</strong>are continually built by a cluster of dedicated machines from the
+<a href="http://cgit.haiku-os.org/haiku/">latest Haiku source code</a> and published every few hours, whereas
+the <strong>stable</strong> builds are intended for everyday use and are released roughly yearly.
+	</p>
+	<p>
+The <strong>nightly</strong> release builds can be found on this website.
+If you wish to use a <strong>stable</strong> release build, you can find the latest stable version
+on <a href="https://www.haiku-os.org/get-haiku/">our main website</a>.
```

Comment:
Wording is a bit meh. You're already on the main website, I suggest the link actually describe what it links to (i.e. stable build).

Reply:
Will work on it, thank you for the feedback.